src/components/atoms/List/List.stories.tsx

```diff
@@ -22,7 +22,7 @@ MultipleElement.args = {
     { id: 1, txt: 'Improve your skills and work on amazing IT projects' },
     {
       id: 2,
-      txt: 'Find support you need and help other people',
+      txt: 'Find support you need and help other people ',
```

Comment:
Is this space really working? I didnt get it to work. My solution for ListItem here https://github.com/Programmers-Only-Group/frontend/pull/41